client/build.gradle.kts

```diff
@@ -37,15 +37,16 @@ wire {
   }
 }
 
-if (rootProject.file("hooks.gradle").exists()) {
-  apply(from = rootProject.file("hooks.gradle"))
+tasks.named("kotlinSourcesJar") {
```

Comment:
is there a non-stringly typed way of doing this dependency?

Reply:
there’s a way that looks less stringly typed:

```
val kotlinSourcesJar by tasks.getting {
  ...
}
```

But that’s potentially worse, because the val name is special